test/modules/test_subdomainenumeration.py

```diff
@@ -0,0 +1,35 @@
+from test.base import ArtemisModuleTestCase
+from typing import NamedTuple
+
+from karton.core import Task
+
+from artemis.binds import TaskType
+from artemis.modules.subdomainenumeration import subdomainenumeration
+
+
+class TestData(NamedTuple):
+    domain: str
+    expected_subdomain: str
+
+
+class SubdomainEnumerationScannerTest(ArtemisModuleTestCase):
+    # The reason for ignoring mypy error is https://github.com/CERT-Polska/karton/issues/201
+    karton_class = subdomainenumeration 
+
+    def test_simple(self) -> None:
```

Comment:
this end-to-end test case makes sense but i'd insist on testing each of get_subdomains_from_... methods